sentry-sidekiq/lib/sentry/sidekiq/cron/job.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
-return unless defined?(::Sidekiq::Cron::Job)
```

Comment:
Hm, but perhaps it's worth keeping the `defined?` check in case `sidekiq-cron` decides to change their API? Would be good to fail gracefully instead of erroring out.

Reply:
In that case they'll need to rename the constant in a way that doesn't fit the convention anymore, which I think is relatively unlikely. And if that do happen, which will be a breaking change to us, we probably don't want it to always fail silently? 🤔